### PR TITLE
Update master

### PR DESCRIPTION
**Pull request by NaMLiM**
Merge the new layout using bootstrap into master branch

### CODE REVIEW

File: css/template-1.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    border-right: 2px #eaba3a solid;
+    width: 70px;
+}
+
+.tanggal-group .hari {
+    font-size: xx-large;
+    font-weight: bolder;
+}
+
+.tanggal-group .tanggal {
```
Comment (NaMLiM) [inline css/template-1.css:77]: Codacy Issue found: [Expected selector ".tanggal-group .tanggal" to come before selector ".popular-post div .tanggal" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 44
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765&bid=17811861)

File: css/template-1.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+.article-berita div h5 a:hover,
+.popular-post div .judul:hover {
+    color: #eaba3a;
+}
+
+/* article berita */
+.tanggal-group {
+    max-height: 90px;
+    border-right: 2px #eaba3a solid;
+    width: 70px;
```
Comment (NaMLiM) [inline css/template-1.css:69]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+.caret-link.scrolled {
+  transform: translateY(10px);
+}
+
+.dropdown:hover .nav-link > .caret-link {
+  transform: translateY(-50px);
+  transition: all 0ms;
+}
+
+.dropdown-menu {
```
Comment (NaMLiM) [inline css/main.css:86]: Codacy Issue found: [Expected selector ".dropdown-menu" to come before selector ".top-layer .dropdown-menu" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+.full h1 dt b {
+  text-align: center;
+}
+
+.header-blog {
+  width: 100%;
+  padding-top: 170px;
+  padding-bottom: 170px;
+  background-attachment: fixed;
+  background: no-repeat center center scroll;
```
Comment (NaMLiM) [inline css/template-2.css:23]: Codacy Issue found: [Unexpected shorthand "background" after "background-attachment" (declaration-block-no-shorthand-property-overrides)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+?>
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>PT. Pindad (Persero) - Home</title>
+        <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+        <?php
+            echo "<link rel='stylesheet' href='css/".$_GET["page"].".css'>";
```
Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$_GET["page"]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+.isi-judul {
+  width: 100%;
+  background-color: #f5f5f5;
+  color: black;
+  padding: 4px 0 4px 0;
+  margin: 2px 0 2px 0;
+}
+
+.isi a,
```
Comment (NaMLiM) [inline css/template-2.css:100]: Codacy Issue found: [Expected selector ".isi a" to come before selector ".list-aside li a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 5 above)
Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php [hunk 7]
```diff
@@ -0,0 +1,311 @@
+<?php
+    if(!$_GET["page"]){
```
Comment (NaMLiM) [inline index.php:2]: Codacy Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+                    <li class="nav-item mx-1">
+                        <a class="nav-link nav-singgle" href="index.php?page=contact-us">HUBUNGI</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <!-- Include Disini -->
+        <?php
+            include("page/".$_GET["page"].".php");
```
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+.isi p a {
```
Comment (NaMLiM) [inline css/template-2.css:101]: Codacy Issue found: [Expected selector ".isi p a" to come before selector ".list-aside li a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.popular-post div .tanggal {
+    font-size: small;
+}
+
+.article-berita div h5 a,
+.popular-post div .judul {
+    text-decoration: none;
+    color: #1e1e1e;
```
Comment (NaMLiM) [inline css/template-1.css:56]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+    font-size: xx-large;
+    font-weight: bolder;
+}
+
+.tanggal-group .tanggal {
+    transform: translate(-2px, -10px);
+}
+
+.article-berita .deskripsi {
+    font-size: small;
```
Comment (NaMLiM) [inline css/template-1.css:82]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+}
```
Comment (NaMLiM) [inline css/template-1.css:83]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 5 above)
Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+.garis {
+  background-color: #eaba3a;
+  width: 40px;
+  height: 5px;
+  position: absolute;
+  transform: translate(-50%, -10px);
+}
+
+
```
Comment (NaMLiM) [inline css/template-1.css:16]: Codacy Issue found: [Expected no more than 1 empty line (max-empty-lines)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+.list-aside li a {
+  text-decoration: none;
+  color: #808080;
+  transition: all 100ms;
+}
+
+.list-aside li a:hover,
+.list-aside li a.active{
```
Comment (NaMLiM) [inline css/template-2.css:59]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+/* article berita */
+.tanggal-group {
+    max-height: 90px;
+    border-right: 2px #eaba3a solid;
+    width: 70px;
+}
+
+.tanggal-group .hari {
+    font-size: xx-large;
+    font-weight: bolder;
```
Comment (NaMLiM) [inline css/template-1.css:74]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+
+.article-berita div h5 a:hover,
+.popular-post div .judul:hover {
+    color: #eaba3a;
+}
+
+/* article berita */
+.tanggal-group {
+    max-height: 90px;
+    border-right: 2px #eaba3a solid;
```
Comment (NaMLiM) [inline css/template-1.css:68]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+    width: 70px;
+}
+
+.tanggal-group .hari {
+    font-size: xx-large;
```
Comment (NaMLiM) [inline css/template-1.css:73]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+  transition: all 300ms;
+}
+
+.to-top.scrolled {
+  display: block;
+}
+
+.to-top:hover {
+  background-color: #1c8596;
+}
```
Comment (NaMLiM) [inline css/main.css:147]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 19]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.article-berita div h5 a:hover,
+.popular-post div .judul:hover {
+    color: #eaba3a;
+}
+
+/* article berita */
+.tanggal-group {
+    max-height: 90px;
```
Comment (NaMLiM) [inline css/template-1.css:67]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+    object-fit: cover;
+}
+
+.popular-post div .tanggal {
+    font-size: small;
+}
+
+.article-berita div h5 a,
+.popular-post div .judul {
+    text-decoration: none;
```
Comment (NaMLiM) [inline css/template-1.css:55]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 21]
```diff
@@ -0,0 +1,83 @@
...
+.img-berita {
+    height: 100%;
+    object-fit: cover;
+}
+
+.popular-post div .tanggal {
+    font-size: small;
+}
+
+.article-berita div h5 a,
```
Comment (NaMLiM) [inline css/template-1.css:53]: Codacy Issue found: [Expected selector ".article-berita div h5 a" to come before selector ".list-aside li:hover > a" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 22]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.list-aside li span {
+  color: #808080;
+  transition: all 100ms;
+  transform: translateX(20px);
+}
+
+.list-aside li:hover > span,
+.list-aside li:hover > a{
```
Comment (NaMLiM) [inline css/template-1.css:39]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 8 above)
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 27
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765&bid=17811861)

File: index.php [hunk 23]
```diff
@@ -0,0 +1,311 @@
+<?php
+    if(!$_GET["page"]){
+        header("location:index.php?page=home");
```
Comment (NaMLiM) [inline index.php:3]: Codacy Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 7 above)
Comment (NaMLiM) [inline index.php:2]: Codacy Issue found: [Processing form data without nonce verification.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 24]
```diff
@@ -0,0 +1,82 @@
...
+.img-berita {
+  height: 100%;
+  object-fit: cover;
+}
+
+.popular-post div .tanggal {
+  font-size: small;
+}
+
+.article-berita div h5 a,
```
Comment (NaMLiM) [inline css/template-1.css:52]: Codacy Issue found: [Expected selector ".article-berita div h5 a" to come before selector ".list-aside li:hover > a" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+.top-layer {
+  z-index: 1040;
+}
+
+.top-layer .dropdown-menu {
+  transform: translate(-100%, -45px);
+  text-overflow: inherit;
+}
+
+.top-layer .dropdown-menu a p{
```
Comment (NaMLiM) [inline css/main.css:23]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 26]
```diff
@@ -0,0 +1,147 @@
...
+  
+.grayLink a:hover {
+  color: #bcbcbc;
+  text-decoration: none;
+}
+  
+.grayColor {
+  color: gray;
+}
+
```
Comment (NaMLiM) [inline css/main.css:130]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
```
Comment (NaMLiM) [inline css/template-2.css:59]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 8 above)
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 28]
```diff
@@ -0,0 +1,103 @@
...
+  transform: translateY(-12px);
+}
+
+.card-img-top {
+  height: 120px;
+  object-fit: cover;
+}
+
+.isi p {
+  color: #777777;
```
Comment (NaMLiM) [inline css/template-2.css:89]: Codacy Issue found: [Expected "#777777" to be "#777" (color-hex-length)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 29]
```diff
@@ -0,0 +1,147 @@
...
+.footer1 {
+  background-color: #03153e;
+}
+  
+.footer2 {
+  background-color: #020e28;
+}
+  
+.grayLink a,
```
Comment (NaMLiM) [inline css/main.css:116]: Codacy Issue found: [Expected selector ".grayLink a" to come before selector ".dropdown-menu a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 8 above)
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: ["include" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required. Concatenating is forbidden. Variables inside are insecure.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 5 above)
Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$_GET["page"]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+.breadcrumb-item {
+  padding-left: 10%;
+}
+
+.breadcrumb-item a {
+  text-decoration: none;
+  color: black;
+}
+
+.breadcrumb-item + .breadcrumb-item::before{
```
Comment (NaMLiM) [inline css/template-2.css:44]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 7 above)
Comment (NaMLiM) [inline index.php:2]: Codacy Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 31]
```diff
@@ -0,0 +1,147 @@
...
+  transition: all 300ms;
+}
+
+.to-top.scrolled {
+  display: block;
+}
+
+.to-top:hover {
+  background-color: #1c8596;
+}
```
Comment (NaMLiM) [inline css/main.css:147]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 32]
```diff
@@ -0,0 +1,103 @@
...
+.isi-judul {
+  width: 100%;
+  background-color: #f5f5f5;
+  color: black;
+  padding: 4px 0;
+  margin: 2px 0;
+}
+
+.isi a,
+.isi p a {
```
Comment (NaMLiM) [inline css/template-2.css:101]: Codacy Issue found: [Expected selector ".isi p a" to come before selector ".list-aside li a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 33]
```diff
@@ -0,0 +1,82 @@
...
+
+.list-aside li span {
+  color: #808080;
+  transition: all 100ms;
+  transform: translateX(20px);
+}
+
+.list-aside li:hover > span,
+.list-aside li:hover > a {
+    color: #eaba3a;
```
Comment (NaMLiM) [inline css/template-1.css:39]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 8 above)
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 5 above)
Comment (NaMLiM) [inline index.php:14]: Codacy Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 7 above)
Comment (NaMLiM) [inline index.php:2]: Codacy Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 34]
```diff
@@ -0,0 +1,103 @@
...
+
+.isi-judul {
+  width: 100%;
+  background-color: #f5f5f5;
+  color: black;
+  padding: 4px 0;
+  margin: 2px 0;
+}
+
+.isi a,
```
Comment (NaMLiM) [inline css/template-2.css:100]: Codacy Issue found: [Expected selector ".isi a" to come before selector ".list-aside li a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-1.css [hunk 35]
```diff
@@ -0,0 +1,82 @@
...
+  border-right: 2px #eaba3a solid;
+  width: 70px;
+}
+
+.tanggal-group .hari {
+  font-size: xx-large;
+  font-weight: bolder;
+}
+
+.tanggal-group .tanggal {
```
Comment (NaMLiM) [inline css/template-1.css:76]: Codacy Issue found: [Expected selector ".tanggal-group .tanggal" to come before selector ".popular-post div .tanggal" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css [hunk 36]
```diff
@@ -0,0 +1,103 @@
...
+.isi p a {
+  color: #eaba3a;
+}
```
Comment (NaMLiM) [inline css/template-2.css:103]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/main.css [hunk 37]
```diff
@@ -0,0 +1,147 @@
...
+.caret-link.scrolled {
+  transform: translateY(10px);
+}
+
+.dropdown:hover .nav-link > .caret-link {
+  transform: translateY(-50px);
+  transition: all 0ms;
+}
+
+.dropdown-menu {
```
Comment (NaMLiM) [inline css/main.css:86]: Codacy Issue found: [Expected selector ".dropdown-menu" to come before selector ".top-layer .dropdown-menu" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: index.php (hunk 8 above)
Comment (NaMLiM) [inline index.php:193]: Codacy Issue found: [Processing form data without nonce verification.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)

File: css/template-2.css (hunk 32 above)
Comment (NaMLiM) [reply css/template-2.css:101]: Ignore

File: css/template-2.css (hunk 34 above)
Comment (NaMLiM) [reply css/template-2.css:100]: Ignore

File: index.php (hunk 8 above)
Comment (NaMLiM) [reply index.php:193]: Ignore

**Comment by kanam129** [comment]
kareh apannah jiah??

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765&bid=17811861)

**Review by kanam129 — APPROVED** [review]
siip

**Pull request merged**